### PR TITLE
Adding the possibility to pull the image from a private repository

### PR DESCRIPTION
**Pull request by grebois**
Adding the possibility to pull the image from a private repository, very useful when using the sentry:onbuild docker image.

### CODE REVIEW

**Comment by grebois** [comment]
@rothgar could you take a look at this?

File: stable/sentry/values.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ image:
   repository: sentry
   tag: "8.20"
```
Comment (rothgar) [inline stable/sentry/values.yaml]: Please bump the default tag to 8.22 to match the chart yaml

**Comment by rothgar** [comment]
This setting also needs to be applied to the db-init and create-user jobs that are run during install. The same base image is applied to each of those jobs but the secret isn't being used so image pulls are failing when I try to use this setting with a private image.

image

**Comment by grebois** [comment]
@rothgar tested locally and looks good now, didn't notice that before, sorry.

**Comment by rothgar** [comment]
No need to apologize. This is why we test.

The db-init and user-create jobs need the secret too
https://github.com/kubernetes/charts/blob/master/stable/sentry/templates/hooks/db-init.job.yaml#L27
https://github.com/kubernetes/charts/blob/master/stable/sentry/templates/hooks/user-create.job.yaml#L26

**Comment by grebois** [comment]
thanks @rothgar, are you able to approve this as well or only @prydonius ?

**Comment by rothgar** [comment]
I'm not an owner so I can't approve but it /lgtm

/meow

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please follow this example:

https://github.com/kubernetes/charts/blob/bc178b63c7df469f1bbccd86629f8cb573a32292/stable/nginx-ingress/templates/default-backend-deployment.yaml#L29-L32

https://github.com/kubernetes/charts/blob/bc178b63c7df469f1bbccd86629f8cb573a32292/stable/nginx-ingress/values.yaml#L341-L342

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/sentry/values.yaml [hunk 2]
```diff
@@ -3,8 +3,11 @@
 # Declare variables to be passed into your templates.
 image:
   repository: sentry
-  tag: "8.20"
+  tag: "8.22"
   pullPolicy: IfNotPresent
+  # Add the secret name to pull from a private registry.
+  imagePullSecrets: []
+  #  name:
```
Comment (unguiculus) [inline stable/sentry/values.yaml]: `# - name:`

Comment (unguiculus) [reply stable/sentry/values.yaml]: Just fixed it myself.

**Review by unguiculus — APPROVED** [review]

**Comment by unguiculus** [comment]
You need to merge in master to make CI happy.

**Comment by grebois** [comment]
@unguiculus merged

**Pull request merged**